app.py

```diff
@@ -16,6 +16,8 @@
 from entity.access_level import restricted
 from mongo_db.index import get_db_collection
 from mongo_db.db_queries import get_all_items, mark_item_as_complete, mark_item_as_uncomplete, mark_item_as_in_progress, add_new_item, remove_item
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
```

Comment:
I don't think this import is used, you always refer directly to the logging module referenced above (e.g. when accessed with `logging.Formatter`, it's the `import logging` rather than this line that's used)